### PR TITLE
 add new blog about slack.

### PR DESCRIPTION
**Pull request by lijiangsheng1**

### CODE REVIEW

File: content/blog/_introduce_slack.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+## Why Slack?
+
+first of all, Slack it cool.
```
Comment (martinyunify) [inline content/blog/_introduce_slack.md:17]: Slack is cool

**Review by chai2010 — COMMENTED** [review]
the reviewer should not change the pr.

@runzexia

**Comment by runzexia** [comment]
please refer to https://github.com/yihui/hugo-xmin/blob/master/exampleSite/content/post/2016-02-14-hello-markdown.md
add title, author, date ... info

**Comment by runzexia** [comment]
`make local` cloud help you with local testing

**Pull request closed without merging**